src/components/DateTimePicker/_date-time-picker.scss

```diff
@@ -0,0 +1,183 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--date-time-picker {
+  padding: 0 $spacing-05 $spacing-05;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+.#{$iot-prefix}--date-time-picker__wrapper {
+  width: 20rem;
+
+  .#{$prefix}--tooltip__trigger.#{$prefix}--tooltip__trigger--definition {
+    font-size: 100%;
+    border-bottom: none;
+    letter-spacing: unset;
+  }
+
+  .#{$prefix}--date-picker--range {
+    position: absolute;
+  }
+
+  .#{$prefix}--date-picker-container {
+    opacity: 0;
+  }
+
+  .flatpickr-calendar.open {
+    position: unset !important;
+    top: unset !important;
+    left: unset !important;
+    box-shadow: none;
+    margin-left: auto;
+    margin-right: auto;
+    -webkit-animation: none;
+    animation: none;
+  }
+
+  .#{$prefix}--number {
+    margin-right: $spacing-05;
+    .#{$prefix}--number__input-wrapper {
+      input {
+        min-width: 8.5rem;
+      }
+    }
+  }
+
+  .#{$prefix}--select-input {
+    width: 8.5rem;
+  }
+
+  .#{$iot-prefix}--time-picker__wrapper {
+    &.#{$iot-prefix}--time-picker__wrapper--with-spinner:first-of-type {
+      margin-right: $spacing-05;
+    }
+    &.#{$iot-prefix}--time-picker__wrapper--with-spinner {
+      .bx--time-picker__input-field {
+        width: 8.5rem;
+        padding-right: $spacing-07;
+      }
+    }
+    .#{$iot-prefix}--time-picker__controls {
+      left: 6.7rem;
+    }
+  }
+
+  .#{$iot-prefix}--date-time-picker__box {
+    outline-offset: -2px;
+    position: relative;
+    list-style: none;
+    display: block;
+    background-color: #ffffff;
+    border: none;
+    border-bottom: 1px solid $carbon--gray-50;
+    width: 100%;
+    height: $spacing-08;
+    cursor: pointer;
+    color: $carbon--gray-100;
+    outline: 2px solid transparent;
+    transition: background-color 70ms cubic-bezier(0.2, 0, 0.38, 0.9);
+    .#{$iot-prefix}--date-time-picker__field {
+      background: none;
+      -webkit-appearance: none;
+      -moz-appearance: none;
+      appearance: none;
+      border: 0;
+      width: 100%;
+      position: relative;
+      display: inline-flex;
+      align-items: center;
+      vertical-align: top;
+      height: calc(100% + 1px);
+      padding: 0 $spacing-09 0 $spacing-05;
+      cursor: pointer;
+      outline: none;
+      white-space: nowrap;
+      & > span {
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+      .#{$iot-prefix}--date-time-picker__icon {
+        position: absolute;
+        top: 0;
+        right: $spacing-05;
+        bottom: 0;
+        height: 100%;
+        transition: -webkit-transform 70ms cubic-bezier(0.2, 0, 0.38, 0.9);
```

Comment:
We should use [the easing functions from `@carbon/motion`](https://github.com/carbon-design-system/carbon/tree/master/packages/motion#sass)

You can `yarn add` it to `dependencies` and **pin the version to 10.5.0.**

@davidicus once this is added, you'll need to update it as well in your WIP branch updating to Carbon v10.10.3 #979

Reply:
perfect! changed to use the duration var :)